src/Contract/Config.purs

```diff
@@ -82,6 +83,26 @@ testnetConfig =
   , hooks: emptyHooks
   }
 
+-- | Blockfrost public preview with CTL as backup
+testnetBlockfrostDevConfig :: Maybe String -> ContractParams
+testnetBlockfrostDevConfig mbApiKey =
+  { backendParams: BlockfrostBackendParams
+      { blockfrostApiKey: mbApiKey
+      , blockfrostConfig: blockfrostPublicPreviewServerConfig
+      }
+      ( Just
+          { ogmiosConfig: defaultOgmiosWsConfig
+          , kupoConfig: defaultKupoServerConfig
+          }
+      )
+  , networkId: TestnetId
+  , walletSpec: Nothing
+  , logLevel: Trace
+  , customLogger: Nothing
+  , suppressLogs: false
+  , hooks: emptyHooks
+  }
```

Comment:
I've tested tx submission and evaluation via this hybrid setup, using CTL for everything else, and it worked fine running the key wallet examples